Client/Ecosia/Extensions/HomepageViewController+Ecosia.swift

```diff
@@ -82,26 +82,6 @@ extension HomepageViewController: NTPLibraryDelegate {
     }
 }
 
-extension HomepageViewController: NTPConfigurableNudgeCardCellDelegate {
-    func nudgeCardRequestToPerformAction(for cardType: HomepageSectionType) {
-        switch cardType {
-        case .newsletterCard:
-            BrazeService.shared.logCustomEvent(.newsletterCardClick)
-            Analytics.shared.newsletterCardExperiment(action: .click)
-            NewsletterCardExperiment.setDismissed()
-        default:
-            return
-        }
-        reloadView()
-    }
-
-    func nudgeCardRequestToDimiss(for cardType: HomepageSectionType) {
-        Analytics.shared.newsletterCardExperiment(action: .dismiss)
-        NewsletterCardExperiment.setDismissed()
```

Comment:
Don't you need to remove `NewsletterCardExperiment` too? I guess this should even break once Core is update with the flag removal.

Reply:
You are right! 
Removed it 🙏